code/controllers/subsystems/atoms.dm

```diff
@@ -8,70 +8,66 @@ SUBSYSTEM_DEF(atoms)
 	init_order = INIT_ORDER_ATOMS
 	flags = SS_NO_FIRE
 
-	var/static/initialized = INITIALIZATION_INSSATOMS
-	// var/list/created_atoms // This is never used, so don't bother. ~Leshana
-	var/static/old_initialized
+	// override and GetArguments() exists for mod-override/downstream hook functionality.
+	// Useful for total-overhaul type modifications.
+	var/adjust_init_arguments = FALSE
 
-	var/list/late_loaders
-	var/list/created_atoms
+	var/atom_init_stage = INITIALIZATION_INSSATOMS
+	var/old_init_stage
 
+	var/list/late_loaders
 	var/list/BadInitializeCalls = list()
 
 /datum/controller/subsystem/atoms/Initialize(timeofday)
 	if(!plant_controller) // Initialize seed repo for /obj/item/seed and /obj/item/weapon/grown
 		plant_controller = new
 	setupgenetics() //to set the mutations' place in structural enzymes, so initializers know where to put mutations.
-	initialized = INITIALIZATION_INNEW_MAPLOAD
-	to_world_log("Initializing objects")
-	admin_notice("<span class='danger'>Initializing objects</span>", R_DEBUG)
+	atom_init_stage = INITIALIZATION_INNEW_MAPLOAD
```

Comment:
Believe this is because of the `(atom_init_stage <= INITIALIZATION_INSSATOMS_LATE)` check in `InitializeAtoms()`, just forcing it to execute regardless of internal state.